code/modules/projectiles/gun.dm

```diff
@@ -169,7 +189,7 @@
 
 
 /obj/item/gun/proc/shoot_live_shot(mob/living/user, pointblank = 0, atom/pbtarget = null, message = 1)
-	if(recoil)
+	if(recoil > 0)
```

Comment:
the current should do this without putting the extra >

Reply:
negative values are truthy, we do not want it triggering on negative values